packages/berry-fslib/sources/NodeFS.ts

```diff
@@ -217,13 +217,13 @@ export class NodeFS extends BasePortableFakeFS {
   }
 
   async readdirPromise(p: PortablePath) {
-    return await new Promise<Array<string>>((resolve, reject) => {
+    return await new Promise<Array<String>>((resolve, reject) => {
```

Comment:
What was the reason to use the class `String` instead of the primitive `string`?

Reply:
Good catch! That's a mistake I'm not used to making, as I'm used to having [a tslint rule](https://github.com/SonarSource/SonarTS/blob/master/sonarts-core/docs/rules/use-primitive-type.md) around to fix this

Reply:
It seems like `typescript-eslint` has an option to [ban some types](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md). We certainly can use it to ban `String` 👍